backend/globaleaks/utils/file_analysis/__init__.py

```diff
@@ -0,0 +1,66 @@
+import logging
+from twisted.internet import abstract
+from globaleaks.models import EnumStateFile
+from globaleaks.rest import errors
+import pyclamd
+
+
+class FileAnalysis:
+    def __init__(self, host='localhost', port=3310):
+        self._host = host
+        self._port = port
+
+    def _scan_file(self, file_name: str, data_bytes: bytes) -> EnumStateFile:
+        logging.info(file_name)
+        try: 
+            cd = pyclamd.ClamdNetworkSocket(self._host, int(self._port))
+            result = cd.scan_stream(data_bytes)
```

Comment:
@alessiofranceschini : still is the usage of a streaming API but externally it will block till all the bytes are processed so this _scan_file function should be run in a deferredThread to not block the process when running it.

In case you will need to fix this, you will find the fix in our merge as we are proceeding fixing this in our integration.